CoreService/Items.psm1

```diff
@@ -156,10 +156,62 @@ Function Get-Item
 	}
 }
 
+function Test-Object
+{
+    <#
+    .Synopsis
+    Checks if the item with the given ID exists.
+	
+	.Notes
+    Example of properties available: Id, Title, etc.
+    
+    For a full list, consult the Content Manager Core Service API Reference Guide documentation 
+    (Tridion.ContentManager.Data.CommunicationManagement.IdentifiableObject object)
+	
+    .Inputs
+    None.
+
+    .Outputs
+    Returns a boolean type.
+
+    .Link
+    Get the latest version of this script from the following URL:
+    https://github.com/pkjaer/tridion-powershell-modules
+
+    .Example
+    Test-Object 'tcm:1-155-5110'
+    Returns if tcm tcm:1-155-5110 exists.
```

Comment:
The uri is actually fine, look in the code, I'm actually testing for tcm
and also for the title. It's really useful
On Mar 21, 2016 11:20 AM, "Peter Kjaer" <notifications@github.com> wrote:

> In CoreService/Items.psm1
> <https://github.com/pkjaer/tridion-powershell-modules/pull/15#discussion_r56795538>
> :
>
> > +    For a full list, consult the Content Manager Core Service API Reference Guide documentation
> > +    (Tridion.ContentManager.Data.CommunicationManagement.IdentifiableObject object)
> > +	
> > +    .Inputs
> > +    None.
> > +
> > +    .Outputs
> > +    Returns a boolean type.
> > +
> > +    .Link
> > +    Get the latest version of this script from the following URL:
> > +    https://github.com/pkjaer/tridion-powershell-modules
> > +
> > +    .Example
> > +    Test-Object 'tcm:1-155-5110'
> > +    Returns if tcm tcm:1-155-5110 exists.
>
> This example uses an invalid URI and the explanation is not grammatically
> correct.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/pkjaer/tridion-powershell-modules/pull/15/files/6df18363718fe61a1c81cd454baa3ecb66312d51#r56795538>
>